src/BloomExe/Browser.cs

```diff
@@ -415,14 +415,37 @@ public void OnGetTroubleShootingInformation(object sender, EventArgs e)
 		public void OnOpenPageInSystemBrowser(object sender, EventArgs e)
 		{
 			Debug.Assert(!InvokeRequired);
-			// An earlier version of this method made a new temp file in hopes that it would go on working
-			// in the browser even after Bloom closed. This has gotten steadily less feasible as we depend
-			// more on the http server. With the <base> element now removed, an independent page file will
-			// have even more missing links. I don't think it's worth making a separate temp file any more.
-			if (Palaso.PlatformUtilities.Platform.IsWindows)
-				Process.Start("Firefox.exe", '"' + _url + '"');
-			else
-				Process.Start("xdg-open", Uri.EscapeUriString(_url));
+			bool isWindows = Palaso.PlatformUtilities.Platform.IsWindows;
+			string genericError = "Something went wrong trying to open this page in ";
+			try
+			{
+				// An earlier version of this method made a new temp file in hopes that it would go on working
+				// in the browser even after Bloom closed. This has gotten steadily less feasible as we depend
+				// more on the http server. With the <base> element now removed, an independent page file will
+				// have even more missing links. I don't think it's worth making a separate temp file any more.
+				if (isWindows)
+					Process.Start("Firefox.exe", '"' + _url + '"');
+				else
+					Process.Start("xdg-open", Uri.EscapeUriString(_url));
+			}
+			catch (Win32Exception)
```

Comment:
Are you sure Linux will throw this specific exception type if there's a problem?